CSharpExtractor/CSharpExtractor/Extractor/Utilities.cs

```diff
@@ -21,11 +22,17 @@ public class Options
         [Option('l', "max_width", Default = 2, HelpText = "Max path length")]
         public int MaxWidth { get; set; }
 
+        [Option('o', "ofile_name", Default = "test.txt", HelpText = "Output file name")]
```

Comment:
This requires a change in the extract.py  script, right?

Reply:
Please take a look at changes in `CSharpExtractor/extract.py` and `preprocess_csharp.sh`